### PR TITLE
Fix --version

### LINKED ISSUE #2431: Executable flag '--version' does not echo correct version
Installed latest version locally via npm.

Expecting '4.3.0', but output is '0.0.1'.

```
$ node -v
v16.13.0

$ npm -v
8.5.5

$ npm i --save-dev  ember-template-lint@latest

$ npm list ember-template-lint
└── ember-template-lint@4.3.0

$ ./node_modules/ember-template-lint/bin/ember-template-lint.js  --version
0.0.1
```

When installed globally, output is 'unknown'

```
$ npm i -g  ember-template-lint@latest
$ ember-template-lint --version
unknown
```

### PR DESCRIPTION
**Pull request by jamescdavis**
fixes #2431

### CODE REVIEW

File: lib/helpers/cli.js [hunk 1]
```diff
@@ -130,7 +132,7 @@ export function parseArgv(_argv) {
     .usage('$0 [options] [files..]')
     .options(specifiedOptions)
     .help()
-    .version();
+    .version(require('../../package.json').version);
```
Comment (bmish) [inline lib/helpers/cli.js:135]: This seems to indicate there is a yargs bug in retrieving the version, perhaps when ESM is present? Is there an active yargs issue for this? I'd prefer it was fixed there. If we are going to add a workaround, we would at least need to comment to explain why it's needed.

Comment (jamescdavis) [reply lib/helpers/cli.js:135]: You're suspicion is correct!: https://github.com/yargs/yargs/issues/2122

Comment (jamescdavis) [reply lib/helpers/cli.js:135]: I've verified https://github.com/yargs/yargs/pull/2123 fixes the issue, so hopefully that will go in soon!

**Comment by jamescdavis** [comment]
This is bug in yargs: https://github.com/yargs/yargs/issues/2122

**Pull request closed without merging**